src/backend/utils/misc/guc_tables.c

```diff
@@ -4491,6 +4492,17 @@ struct config_string ConfigureNamesString[] =
 		check_canonical_path, NULL, NULL
 	},
 
+	{
+		{"extension_destdir", PGC_SUSET, FILE_LOCATIONS,
```

Comment:
`dynamic_library_path` is `CLIENT_CONN_OTHER` not `FILE_LOCATIONS`. Should this be the same for consistency, and is there a security reason for the dynamic library path choice?

Reply:
It's superuser-only, so I'm not sure it matters. Happy to change it to whatever the consensus determines.